website/members/models/member.py

```diff
@@ -168,6 +168,13 @@ def can_attend_events(self):
             and self.current_membership is not None
         )
 
+    @property
+    def can_attend_alumni_events(self):
```

Comment:
Looks good, I only wouldn't call this `alumni events` as that sounds quite restricting. I would call it `can_attend_events_without_membership`